nannou/src/wgpu/render_pipeline_builder.rs

```diff
@@ -237,16 +240,20 @@ impl<'a> RenderPipelineBuilder<'a> {
     }
 
     /// The color blending used for this pipeline.
-    pub fn color_blend(mut self, blend: wgpu::BlendState) -> Self {
+    pub fn color_blend(mut self, color_blend: wgpu::BlendComponent) -> Self {
         let state = self.color_state.get_or_insert(Self::DEFAULT_COLOR_STATE);
-        state.color_blend = blend;
+        if let Some(blend) = state.blend.as_mut() {
+            blend.color = color_blend;
+        }
```

Comment:
I think this should probably be:

```rust
let state = self.color_state.get_or_insert(Self::DEFAULT_COLOR_STATE);
let blend = state.blend.get_or_insert(Self::DEFAULT_BLEND_STATE);
blend.color = color_blend;
```

In practise it would have been fine as `state.blend` is always `Some` due to the default, however if it were ever changed to `None` and this was called, the specified `color_blend` would be ignored.

Reply:
That's a good point. I adjusted it